lisa/schema.py

```diff
@@ -565,6 +565,65 @@ def _call_requirement_method(self, method_name: str, capability: Any) -> Any:
         return value
 
 
+@dataclass_json()
+@dataclass()
+class ACCOptionSettings(FeatureSettings):
+    type: str = "ACC"
+    is_supported: Optional[bool] = None
+
+    def __eq__(self, o: object) -> bool:
+        assert isinstance(o, ACCOptionSettings), f"actual: {type(o)}"
+        return self.is_supported == o.is_supported
+
+    def __str__(self) -> str:
+        return self.__repr__()
+
+    def __repr__(self) -> str:
+        return f"is_supported: {self.is_supported}"
+
+    def check(self, capability: Any) -> search_space.ResultReason:
+        assert isinstance(capability, ACCOptionSettings), f"actual: {type(capability)}"
+        result = search_space.ResultReason()
+        result.merge(
+            self._check_acc(self.is_supported, capability.is_supported),
+            "is_supported",
+        )
+
+        return result
+
+    def _call_requirement_method(self, method_name: str, capability: Any) -> Any:
+        assert isinstance(capability, ACCOptionSettings), f"actual: {type(capability)}"
+        parent_value = super()._call_requirement_method(method_name, capability)
+
+        # convert parent type to child type
+        value = ACCOptionSettings()
+        value.extended_schemas = parent_value.extended_schemas
+        value.is_supported = capability.is_supported
+
+        return value
+
+    def _check_acc(
+        self, requirement: Optional[bool], capability: Optional[bool]
+    ) -> search_space.ResultReason:
+        result = search_space.ResultReason()
+        # if requirement is none, capability can be either of True or False
+        # else requirement should match capability
+        if requirement is not None:
```

Comment:
The logic may not need to be so complex. But let's verify if the "is_supported" can be removed. If it can be removed, follow other features pattern, they would be simpler.